### PR TITLE
added the first task

### PR DESCRIPTION
**Pull request by jaykapadiya2004**

### CODE REVIEW

File: .DS_Store
Comment (ialimustufa) [inline .DS_Store:1]: Remove this file.

File: student-projects/.DS_Store
Comment (ialimustufa) [inline student-projects/.DS_Store:1]: Remove this file.

File: student-projects/day1/.DS_Store
Comment (ialimustufa) [inline student-projects/day1/.DS_Store:1]: Remove this file.

File: student-projects/.DS_Store
Comment (jaykapadiya2004) [reply student-projects/.DS_Store:1]: ok

**Pull request closed without merging**